gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Project-wide Gradle settings.
-VERSION_NAME=5.1.0-SNAPSHOT
+VERSION_NAME=5.4.1-SNAPSHOT
```

Comment:
5.4.1 is the latest Mapbox Java version, so wouldn't this be `5.5.0-SNAPSHOT`? 🤔

Reply:
yeah, seems that CI requires it https://github.com/mapbox/mapbox-java/blob/master/circle.yml#L81-L87 🤔  gonna create another PR

Reply:
@langsmith https://github.com/mapbox/mapbox-java/pull/1173 👀

Reply:
👍